CHANGELOG.md

```diff
@@ -146,6 +146,8 @@
 
 ## 5.21.0 - 2020-05-26
 
+## 5.16.0 - 2019-09-29
```

Comment:
Thanks, @x-limitless-x, there is something wrong with the changelog. Can you revert this file in your fork, please?

Reply:
I don't know how to do that 😢

Reply:
Ok, no problem. You just change the file another time like you did before. Then you push the change to your fork, also like you have pushed your previous changes. This pull request will be automatically updated. Just give it a try, it is hard to do something wrong. And if something bad happens, git let's you always travel back in time.

Reply:
Just updated, should be good to go now 👍